chia/wallet/wallet_state_manager.py

```diff
@@ -83,6 +83,19 @@
 from chia.wallet.wallet_user_store import WalletUserStore
 
 
+# this is a placeholder function to bridge wallets that don't yet have
+# the `puzzle_hash_for_pk` optimization in them
```

Comment:
If we were to define a Protocol for wallets, and require wallets to implement it. Would mypy be able to tell us about wallets not implementing new functions like this at build-time?

Reply:
Yes, for sure. I think it's a good idea, but it's probably out of scope of this patch (and I wouldn't call myself an expert on this wallet code).